admin/app/dfp/DfpDataCacheJob.scala

```diff
@@ -44,11 +42,7 @@ class DfpDataCacheJob(capiLookupAgent: CapiLookupAgent) extends ExecutionContext
       _ <- CustomTargetingValueAgent.refresh()
       _ <- PlacementAgent.refresh()
     } {
-      val data = loadLineItems()
-      val paidForTags = paidForTagsFinder.fromLineItems(data.lineItems)
-      capiLookupAgent.refresh(paidForTags) map {
-        _ => write(data)
-      }
+      loadLineItems()
```

Comment:
I think there are more reading jobs since this code was written, so probably not.